db/migrations/20200226203241_add_name_to_movies.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+exports.up = function (Knex) {
+  return Knex.schema.table('movies', (table) => {
+    table.text('name');
+  }).then(() => {
```

Comment:
at this point, because most of our node repos are node 10/12+, can you trying using `async await` here too? it's just a good practice going forwards whenever possible.

Reply:
+1